.changeset/nine-lies-press.md

```diff
@@ -0,0 +1,7 @@
+---
+"@nomicfoundation/hardhat-verify": patch
```

Comment:
Only the tests were changed in `hardhat-verify`, it shouldn't be included in the changeset.